diff --git a/lib/python/qti/aisw/converters/backend/custom_ops/op_factory.py.bak b/lib/python/qti/aisw/converters/backend/custom_ops/op_factory.py
index 2c64865..198cbd2 100644
--- a/lib/python/qti/aisw/converters/backend/custom_ops/op_factory.py.bak
+++ b/lib/python/qti/aisw/converters/backend/custom_ops/op_factory.py
@@ -13,6 +13,7 @@ class UDOFactory(CustomOpFactory, metaclass=ABCMeta):
     # global variables
     package_resolver = dict()
     op_collection = CustomOpCollection()
+    default_op_collection = []
 
     def __init__(self):
         super(UDOFactory, self).__init__()
